styleguide/public/styleguide/img/svg/accolade.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="36" height="19" viewBox="0 0 36 19">
```

Comment:
deze file is nergens gebruikt, toch?

Reply:
Neen, we gebruiken echter wel de svg code ervan, dus vondt ik het wel handig de source image die uit Sketch komt mee in de repo te steken, in het geval er zaken veranderen.